audformat/core/testing.py

```diff
@@ -34,6 +34,52 @@
 )
 
 
+def add_misc_table(
+        db: Database,
+        table_id: str,
+        index: pd.Index,
+        *,
+        columns: Union[
+            str,
+            Sequence[str],
+            Dict[str, Union[
+                str, Tuple[Optional[str], Optional[str]]
+            ]],
+        ] = None,
+        p_none: float = None,
+        split_id: str = None,
+        media_id: str = None,
+) -> MiscTable:
+    r"""Add a miscellaneous table with random values.
+
+    By default, adds one column for every scheme in the database.
+    To create a specific set of columns use ``columns``.
+
+    Args:
+        db: a database
+        table_id: ID of table that will be created
+        index: index object
+        columns: a list of scheme_ids or a dictionary with column names as
+            keys and tuples of ``(scheme_id, rater_id)`` as values. ``None``
+            values are allowed
+        p_none: probability to draw invalid values
```

Comment:
This I do not get completely, how can we assign values that do not match the scheme without getting an error?

Reply:
With `invalid` we refer to `None` values. I will rephrase.

Reply:
done

Reply:
Ah, this makes indeed more sense, thanks.